middleware/session/store.go

```diff
@@ -43,19 +43,18 @@ func (s *Store) Get(c *fiber.Ctx) (*Session, error) {
 	loadData := true
 
 	id := s.getSessionID(c)
-
-	if len(id) == 0 {
+	if id == "" {
 		fresh = true
 		var err error
 		if id, err = s.responseCookies(c); err != nil {
 			return nil, err
 		}
-	}
 
-	// If no key exist, create new one
-	if len(id) == 0 {
-		loadData = false
-		id = s.KeyGenerator()
+		// If key still doesn't exist, create new one
```

Comment:
the old way of writing is flatter and offers no disadvantages to the new way of writing

please change back

Reply:
It's doing the `if id == ""` check twice if `id` is non-empty in the first run, so it's speeding up the code a little bit.